recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7796-salvator-x-domd.dts

```diff
@@ -36,15 +36,20 @@
 #include <dt-bindings/gpio/gpio.h>
 
 / {
-	passthrough {
-	};
 
 	aliases {
-		serial0 = &scif2;
 		serial1 = &scif1;
 		ethernet0 = &avb;
 	};
 
+	soc {
+		//this must point to Xen interrupt-parent which is 0xfde8
+		interrupt-parent = <0xfde8>;
```

Comment:
Looks like it is not a worldwide constant.  Can't we insert the proper value in an automatic way?

Reply:
I see no way to do so in a generic way:
tools/libxl/libxl_arm.c:
/*
 * The device tree compiler (DTC) is allocating the phandle from 1 to
 * onwards. Reserve a high value for the GIC phandle.
 */
#define PHANDLE_GIC (65000)

Reply:
```diff --git a/recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7796-salvator-x-domd.dts b/recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7796-salvator-x-domd.dts
index 3316682e5616..30274ecfa76f 100644
--- a/recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7796-salvator-x-domd.dts
+++ b/recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7796-salvator-x-domd.dts
@@ -35,6 +35,12 @@
 #include "r8a7796.dtsi"
 #include <dt-bindings/gpio/gpio.h>
 
+/*
+ * The device tree compiler (DTC) is allocating the phandle from 1 to
+ * onwards. Reserve a high value for the GIC phandle.
+ */
+#define PHANDLE_GIC (65000)
+
 / {
 
        aliases {
@@ -43,8 +49,8 @@
        };
 
        soc {
-               //this must point to Xen interrupt-parent which is 0xfde8
-               interrupt-parent = <0xfde8>;
+               //this must point to Xen interrupt-parent
+               interrupt-parent = <PHANDLE_GIC>;
        };
 
        passthrough {
```